### ISSUE TITLE
no VST plugins available

### ISSUE THREAD

**Issue by grammoboy2** [kmatheussen/radium#1318]
I'm not sure what happened, but after I installed version 6.5.85 from git tag, I don't have VST plugins available. Even if I add the PATHs and rescan.

Ok something with the new build you would think, but also the latest demo version doesn't give me the VST plugins anymore. 

This morning/ yesterday, they where available. In other apps (Carla) VST plugins still works fine.

**Comment by kmatheussen** [comment]
That's strange. They work in 6.5.84, but not in 6.5.85?

**Comment by grammoboy2** [comment]
I had it working in a demo version at least, but also in the latest git version. 
But now not in version 6.5.85 (git) and the latest demo version. Strange.

**Comment by kmatheussen** [comment]
It would be enormously helpful if you could try more of the demo versions
and pinpoint exactly when it stopped working...
(don't you have access to the full version binaries by the way?).

Also, can you name one VST plugin that doesn't work now? Maybe I just don't
have the VST plugins you have.

**Comment by kmatheussen** [comment]
Maybe it works if you delete all the plugin-info files in
~/.radium/scanned_plugins/:
rm -f ~/.radium/scanned_plugins/*

**Comment by grammoboy2** [comment]
I had the latest git installed with working VST. Then decided on your recommendation to install the stable version. Tried to build and install git tag 6.5.86 several times.
After trying everything I could to get it running, I tried tag 6.5.85 instead. That worked. Then I got a message that it couldn't load Zynaddsubfx VST in the session I had running this morning.

Then I checked the demo version, but now there wheren't VSTs either anymore, while it worked yesterday.

I'll check further tomorrow.

Zynaddsubfx VST
Nekobee VST
are examples I have and tried.
